### DISCUSSION TITLE
Is it possible to disable cache of hits?

### DISCUSSION THREAD

**Discussion by thedevvikas** [searchkit/searchkit#1258] [Q&A] [answered]
Hi everyone, I want searchkit to do `_msearch` request on every pagination. Suppose I have 3 pages of data and currently I am on page 1 (data gets fetched)...if I paginate to page 2 (data gets fetched again)...BUT when I go back to page 1, it returns hits from cache. Instead, I want searchkit to re-fetch data from server. Is this possible ? Thanks. 😊 

I am using following packages:
`"@searchkit/instantsearch-client": "^4.4.0"`
`"react-instantsearch-hooks-web": "^6.41.0"`
`"searchkit": "^4.4.0"`

**Comment by joemcelroy** [comment, marked as answer]
Hey! Its not possible to disable unfortunately - tracking issue here https://github.com/searchkit/searchkit/issues/1255

Looks like not possible to do it either with hooks-web. 

You could also call clearCache method on the [@searchkit/instantsearch-client ](https://github.com/searchkit/searchkit/blob/main/packages/searchkit-instantsearch-client/src/index.ts#L28)

I will add a configuration in instantsearch-client to not cache soon!

**Reply by thedevvikas** [reply]
Thanks @joemcelroy ! 👍
